txpool/txpool.go

```diff
@@ -427,11 +427,14 @@ func (p *TxPool) Drop(tx *types.Transaction) {
 	clearAccountQueue(dropped)
 
 	p.eventManager.signalEvent(proto.EventType_DROPPED, tx.Hash)
-	p.logger.Debug("dropped account txs",
-		"num", droppedCount,
-		"next_nonce", nextNonce,
-		"address", tx.From.String(),
-	)
+
+	if p.logger.IsDebug() {
+		p.logger.Debug("dropped account txs",
+			"num", droppedCount,
+			"next_nonce", nextNonce,
+			"address", tx.From.String(),
```

Comment:
I think we do not need to explicitly convert `tx.From.String()`. We can just use `tx.From` and in that case even if is unnecessary

Reply:
@igorcrevar We could, but in that case the value will be converted to String by using reflection in the logger code, which is much slower. I considered that approach but eventually opted for this one. But I could change it if we agree it is better to have it that way.